source/test/eslint-compatibility.ts

```diff
@@ -0,0 +1,29 @@
+import * as path from 'path';
+import test from 'ava';
+import * as execa from 'execa';
+
+test('`expectType` is compatible with eslint @typescript-eslint/no-unsafe-call rule', t => {
+	try {
+		execa.sync(
+			'node_modules/.bin/eslint',
+			['source/test/fixtures/eslint-compatibility', '--no-ignore'],
+			{
+				cwd: path.join(__dirname, '../../')
+			}
+		);
+
+		t.fail('eslint should have found an error!');
+	} catch (e: unknown) {
```

Comment:
```suggestion
	} catch (error: unknown) {
```

You can use `error_` or `error2` for the next one.

Reply:
done